state-chain/runtime/src/lib.rs

```diff
@@ -1118,6 +1120,76 @@ impl frame_support::traits::OnRuntimeUpgrade for FlipToBurnMigration {
 	}
 }
 
+pub struct ActiveBidderMigration;
```

Comment:
Please add this in its own module in `migrations`. The runtime.rs is big enough already, and adding it in a separate module make it easier to resolve conflicts when migrations are added/removed.